src/main/java/io/muehlbachler/bswe3b/CalculatorService.java

```diff
@@ -5,9 +5,14 @@
 // calculator - documentation n
 @Service
 public class CalculatorService {
-    // divide - should return a/b
-    public int divide(final int a, final int b) {
-        // FIXME: implement
-        return -1;
+   public int divide(final int a, final int b) {
+        if (b == 0) {
```

Comment:
the spec was if b <= 0, return 0.

so this would not adhere to those specs BUT if you want to throw an exception you should test for that case as well. you can use `assertThrows` for that: https://junit.org/junit5/docs/5.8.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertThrows(java.lang.Class,org.junit.jupiter.api.function.Executable).